ksql-package/src/assembly/package.xml

```diff
@@ -0,0 +1,45 @@
+<assembly xmlns="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.2"
+          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:schemaLocation="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.2
+          http://maven.apache.org/xsd/assembly-1.1.2.xsd">
+    <!-- Assembles a package that can run ksql -->
+    <id>package</id>
+    <formats>
+        <format>dir</format>
+        <format>tgz</format>
+        <format>zip</format>
+    </formats>
+    <includeBaseDirectory>true</includeBaseDirectory>
+    <baseDirectory>ksql</baseDirectory>
+    <fileSets>
+        <fileSet>
+            <directory>${project.parent.basedir}</directory>
+            <outputDirectory></outputDirectory>
+            <includes>
+                <include>README*</include>
+                <include>LICENSE*</include>
+                <include>ksq-lrocket.png</include>
+                <include>screencast.jpg</include>
+                <include>licenses/</include>
+                <include>notices/</include>
+                <include>bin/</include>
+                <include>config/</include>
+                <include>docs/</include>
+                <include>ksql-clickstream-demo/</include>
```

Comment:
Is this needed in the package? A standalone download for the demo might make sense.

Reply:
I added it because the docs had pointers to it that I didn't want to be broken. But the docs aren't rendering properly anyway so I just removed all the docs stuff from the binary (clickstream included)